### PR TITLE
Acceptance tests

### PR DESCRIPTION
**Pull request by bobbyiliev**
Adding more acceptance tests for the following resources:

- [x] Views
- [x] Tables
- [x] Types
- [x] Schemas
- [x] Load generator source
- [x] Sink Kafka
- [x] Source Kafka	
- [x] Source Postgres

### CODE REVIEW

File: pkg/provider/acceptance_schema_test.go [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:          func() { testAccPreCheck(t) },
+		ProviderFactories: testAccProviderFactories,
+		CheckDestroy:      nil,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccSchemaResource(schemaName),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckSchemaExists("materialize_schema.test"),
+					resource.TestCheckResourceAttr("materialize_schema.test", "name", schemaName),
```
Comment (dehume) [inline pkg/provider/acceptance_schema_test.go:26]: A couple more attributes to include `database_name` and `qualified_sql_name`

File: pkg/provider/acceptance_table_test.go [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:          func() { testAccPreCheck(t) },
+		ProviderFactories: testAccProviderFactories,
+		CheckDestroy:      nil,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccTableResource(tableName),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckTableExists("materialize_table.test"),
+					resource.TestCheckResourceAttr("materialize_table.test", "name", tableName),
```
Comment (dehume) [inline pkg/provider/acceptance_table_test.go:26]: There are a few other attributes we should track like the `schema_name`, `database_name`, `qualified_sql_name`, `column`

File: pkg/provider/acceptance_type_test.go [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:          func() { testAccPreCheck(t) },
+		ProviderFactories: testAccProviderFactories,
+		CheckDestroy:      nil,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccTypeResource(typeName),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckTypeExists("materialize_type.test"),
+					resource.TestCheckResourceAttr("materialize_type.test", "name", typeName),
```
Comment (dehume) [inline pkg/provider/acceptance_type_test.go:26]: There are a few other attributes we should track like the `schema_name`, `database_name`, `qualified_sql_name`, `list_properties`, `map_properties`, `category`

File: pkg/provider/acceptance_view_test.go [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:          func() { testAccPreCheck(t) },
+		ProviderFactories: testAccProviderFactories,
+		CheckDestroy:      nil,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccViewResource(viewName),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckViewExists("materialize_view.test"),
+					resource.TestCheckResourceAttr("materialize_view.test", "name", viewName),
```
Comment (dehume) [inline pkg/provider/acceptance_view_test.go:26]: There are a few other attributes we should track like the `schema_name`, `database_name`, `qualified_sql_name`, `statement`

**Review by dehume — CHANGES_REQUESTED** [review]
Overall looks good. Only thing is some of these resources have additional attributes we can check in state after the apply.

File: pkg/provider/acceptance_source_load_generator_test.go [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:          func() { testAccPreCheck(t) },
+		ProviderFactories: testAccProviderFactories,
+		CheckDestroy:      nil,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccSourceLoadGeneratorResource(sourceLoadGeneratorName),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckSourceLoadGeneratorExists("materialize_source_load_generator.test"),
+					resource.TestCheckResourceAttr("materialize_source_load_generator.test", "name", sourceLoadGeneratorName),
```
[the remark this answers is not in the document]
Comment (bobbyiliev) [reply pkg/provider/acceptance_source_load_generator_test.go:26]: Goot catch! Just added the missing attributes for all of those resources! Let me know if you notice any other issues!

**Review by dehume — APPROVED** [review]

**Pull request merged**